heimdall-gateway/src/main/java/br/com/conductor/heimdall/gateway/filter/HeimdallDecorationFilter.java

```diff
@@ -249,76 +263,68 @@ protected HeimdallRoute getMatchingHeimdallRoute(String requestURI, String metho
 
         boolean auxMatch = false;
         for (Entry<String, ZuulRoute> entry : routeLocator.getAtomicRoutes().get().entrySet()) {
-            if (entry.getKey() != null) {
-                String pattern = entry.getKey();
```

Comment:
Keep pattern variable. This variable makes the code more legible, because I don't know what is inside of entry.getKey() and the variable name show it to me